src/tools.c

```diff
@@ -215,6 +215,23 @@ void MakeDir(char file[128], const char *path, const char *dir, const char
 bool MakeFileName(char file[128], const char *path, const char *dir, const
     char *sdir, const char *name, const char *extens)
 {  FILE *fp;
+   char name2[8];  /* AYM 1999-01-13: keep checker happy */
+   /* deal with VILE strange name
+   ** replace the VILE\ 
+   ** by          VIL@B 
+   */
+   Normalise(name2,name);
+ 
+   /* 2017-07-17: Only replace backslash. [ and ] characters are 
+      supported in Windows and Unix-derived systems.  */
```

Comment:
Don't bother dating your comment, Git does that just fine automatically ;)

The comment may better just be something like "Replace backslash which is an invalid character on Windows systems." -- no mention of [ or ] are needed.